### DISCUSSION TITLE
Show disaggregated data when JS is off

### DISCUSSION THREAD

**Discussion by LucyGwilliamAdmin** [open-sdg/open-sdg#1609] [Ideas]
When JS is off, users have no control over viewing disaggregated data online - instead they need to download the source CSV.

**To discuss: should we/is it possible to display all data when JS is off? Is MVP to let users know there is disaggregated data available in data download?**

Indicator page when JS is on:
<img src="https://user-images.githubusercontent.com/43404358/168613270-9221ef74-c5a4-4422-94b4-07f68ee5ceb3.png" height="75%" width="75%">

Indicator page when JS is off:
<img src="https://user-images.githubusercontent.com/43404358/168613516-dfee8762-d244-4d3a-bdbe-6606d1ad0313.png" height="75%" width="75%">

**Comment by LucyGwilliamAdmin** [comment]
@phillipgardiner @brockfanning @otis-bath 
Thoughts please! Personally I think it would be overkill to make all disaggregations viewable by default when JS is off - table would be massive, especially where lots of disaggregations are provided.

Instead I think we should add somewhere that 'disaggregations X, Y and Z are available in data download' - could this be automated?

**Comment by brockfanning** [comment]
I don't know offhand if it would be possible to list the disaggregations in the message, but my rough prediction would be that it would be difficult. I know that a more generic message like "Additional disaggregations are available in data download" is possible, and can be hidden if javascript is on.

**Reply by phillipgardiner** [reply]
Yep, I read something yesterday that said using a message like this is the approach people tend to use.
